test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -86,19 +111,33 @@ def func(x):
 
         _run_test_with_fx_to_onnx_exporter_reference_runtime(func, (tensor_x,))
 
-    @unittest.skip("TypeError: export() got an unexpected keyword argument 'b'")
     def test_func_with_args_and_kwargs(self):
-        def func(x, b=1.0):
+        # Non-tensor optional kwargs are always folded into constant and
+        # removed from input list in Dynamo-traced graph, so we can't
+        # define a function like
+        #   def func(x, b=1.0)
+        # here. E.g., if you change the `b` to 1.0 below, it will complain
+        # somewhere that model is called with extra args because the modified
+        # function is traced into
+        #   def forward(self, x : torch.Tensor):
+        #     add = x + 1.0;  x = None
+        #     relu = add.relu()
+        #     return (add, relu)
+        # To summary, optional kwargs must be tensors; otherwise, they are
+        # treated as in-graph constants in Dynamo.
+        def func(x, b=torch.tensor(1.0)):
             y = x + b
             z = y.relu()
             return (y, z)
 
         tensor_x = torch.randn(1, 1, 2, dtype=torch.float32)
 
-        # This is the only call to verification.verify_model_with_fx_to_onnx_exporter,
-        # which introduces dependency of onnxscript to torch.
-        # Commenting this line and removing related files.
-        # self.run_test_with_fx_to_onnx_exporter(func, (tensor_x,), {"b": 500.0})
+        # Test without providing optional kwarg.
+        _run_test_with_fx_to_onnx_exporter_reference_runtime(func, (tensor_x,))
```

Comment:
Do we want to run this with ort too?

Reply:
As long as it works, I don't have preference. Any suggested function I should use here?

Reply:
looks like it's just a naming issue. I commented inline.